admin/app/components/solidus_admin/orders/index/component.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="<%= stimulus_id %> px-4">
+  <header class="py-6 flex items-center">
+    <h1 class="body-title">
+      <%= title %>
+    </h1>
+
+    <div class="ml-auto flex gap-2 items-center">
+      <span class="max-w-xs text-xs text-right text-gray-600 hidden lg:block">
+        <%= t(".feedback_description") %>
+      </span>
+      <%= render @button_component.new(
+        tag: :a,
+        text: t(".give_feedback"),
+        href: "https://solidus.io/feedback?category=solidus-admin",
+        icon: "feedback-line",
+        scheme: :secondary,
+        target: :_blank,
+      ) %>
```

Comment:
Given that we'll need this button for every page, what do you think about extracting it into a component?

Reply:
Thanks for the suggestion! I've refactored it into a `ui/feedback` component. However, I'm on the fence about classifying it strictly as a UI component, so let me know what you think 👍

Reply:
Yeah, I don't see it as a UI thing. Probably it should be a top-level component, just like the skip link. Maybe at some point we can consider grouping non-UI components in a certain way?